### DISCUSSION TITLE
Question about the Ping module in go.d.plugin

### DISCUSSION THREAD

**Discussion by olg33** [netdata/netdata#17210] [Q&A]
Hello everyone,

I have just  installed the Ping module and it works fine, but the VM where I installed this module has multiple network interfaces and I need to ping elements connected to these other interfaces. I just went through the manual at https://learn.netdata.cloud/docs/collecting-metrics/synthetic-checks/ping and I only could find how to configure the host to ping, like this: 

 `jobs:`
  `    - name: Network Int 1`
     `      hosts:                 `
          `         - "www.google.com"`

Is there any way to indicate which network interface should the module use to ping a remote host? 

I'd appreciate any input on this matter.

**Comment by ilyam8** [comment]
Hi, @olg33. Do you mean the output interface? No, it is not possible. There is a [PR that adds this functionality](https://github.com/prometheus-community/pro-bing/pull/32) to the upstream library that Netdata uses for pinging, but it has not been merged.

**Reply by olg33** [reply]
Thanks a lot  Ilyam, sorry, but I'm not very proficient with Netdata implementation,  Is there any document or link that describes how to implement this PR? I saw the link you included but I'm still not sure how to implement it in my Netdata VM. If it's too complicated, maybe I should just consider that this feature is not available.

**Reply by ilyam8** [reply]
Yes, the feature is not available at the moment.

**Reply by ilyam8** [reply]
Added in #19016
